README.md

```diff
@@ -1,5 +1,9 @@
 #fh-messaging(1) -- The FeedHenry Message Server
```

Comment:
@camilamacedo86 this change looks fine. Just a small note that the first header is not rendering as there is a space missing after the #. Could that be updated as part of this pr also?

Reply:
Done:-)